test/adoption_center/test_client.rb

```diff
@@ -0,0 +1,18 @@
+require 'minitest/autorun'
```

Comment:
This probably belongs in the `test_helper.rb` file which is the one that should be required here.